src/pveclib/vec_int64_ppc.h

```diff
@@ -3230,12 +3235,12 @@ vec_stvsudo (vui64_t xs, unsigned long long int *array,
  *
  *  @param xs Vector of doubleword elements to store.
  *  @param array Pointer to array of unsigned doublewords.
- *  @param vra Vector of doubleword indexes.
- *  @param scale Factor effectually multiplying the indexes by
+ *  @param vra Vector of doubleword indices.
+ *  @param scale 8-bit integer. Indexes are multiplying by
```

Comment:
s/multiplying/multiplied/

Reply:
Ok, still working through the name change. Plus Clang (stxsdx?) and P7 (no vec_vsld for index shift). When I have f64 changes complete. Then I will decide to merge this or do a new int64 pull.